Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -129,7 +131,14 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		ini.GetIfExists("EmuState", "EmulationIssues", &m_issues);
 	}
 
-	if (!m_pImage.empty())
+	if (IsElfOrDol())
```

Comment:
Check `!IsValid()` or `!pVolume` before checking `IsElfOrDol()`, otherwise weird things will happen if someone has a valid volume that has the file name extension .dol or .elf.